<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInjectDll">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>InjectDll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSendCloseMsg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SendCloseMsg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUnSetHook">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>UnSetHook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetHookSelf">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>110</y>
     <width>97</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SetHookSelf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetHook">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SetHook</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>252</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Hwnd(hex)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEditHwnd"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>451</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>dll path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdllPath"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadHook">
      <property name="text">
       <string>LoadHook</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>204</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>pid</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdpid"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnTest1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>test1</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
